oeplatform/settings.py

```diff
@@ -88,6 +89,15 @@
     }
 ]
 
+CORS_ORIGIN_WHITELIST = [
```

Comment:
this is not used anymore, isnt it? please make sure you dont leave deprecated code here :) Ignore if im wrong